geode-wan/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeRegionDUnitTest.java

```diff
@@ -0,0 +1,165 @@
+    MemberVM server2 = lsRule.startServerVM(4, "group2", sending_locator.getPort());
+
+    configureServers(server1, server2);
+
+    gfsh.connectAndVerify(sending_locator);
+    gfsh.executeAndAssertThat("create async-event-queue --id=queue1 --group=group1 "
+        + "--listener=org.apache.geode.internal.cache.wan.MyAsyncEventListener").statusIsSuccess();
+    gfsh.executeAndAssertThat("create gateway-sender --id=sender1 --remote-distributed-system-id=2")
+        .statusIsSuccess();
+    sending_locator.waitTillAsyncEventQueuesAreReadyOnServers("queue1", 1);
+    sending_locator.waitTilGatewaySendersAreReady(2);
+
+    gfsh.executeAndAssertThat(
+        "create region --name=region4 --type=REPLICATE --async-event-queue-id=queue1 --gateway-sender-id=sender1")
+        .statusIsSuccess();
+
+  }
+
+  @SuppressWarnings("deprecation")
+  private static void configureServers(MemberVM server1, MemberVM server2) {
+    server1.invoke(() -> {
+      final Cache cache = LocatorServerStartupRule.getCache();
+      RegionFactory<String, Integer> dataRegionFactory =
+          cache.createRegionFactory(RegionShortcut.PARTITION);
+      dataRegionFactory.setConcurrencyLevel(4);
+      EvictionAttributes ea =
+          EvictionAttributes.createLIFOEntryAttributes(100, EvictionAction.LOCAL_DESTROY);
+      dataRegionFactory.setEvictionAttributes(ea);
+      dataRegionFactory.setEnableAsyncConflation(true);
+
+      FixedPartitionAttributes fpa =
+          FixedPartitionAttributes.createFixedPartition(PART1_NAME, true);
+      PartitionAttributes pa = new PartitionAttributesFactory().setLocalMaxMemory(100)
+          .setRecoveryDelay(2).setTotalMaxMemory(200).setRedundantCopies(1)
+          .addFixedPartitionAttributes(fpa).create();
+      dataRegionFactory.setPartitionAttributes(pa);
+
+      dataRegionFactory.create(PR1);
+      createLocalRegion(LOCAL_REGION);
+    });
+
+    server2.invoke(() -> {
+      final Cache cache = LocatorServerStartupRule.getCache();
+      RegionFactory<String, Integer> dataRegionFactory =
+          cache.createRegionFactory(RegionShortcut.PARTITION);
+      dataRegionFactory.setConcurrencyLevel(4);
+      EvictionAttributes ea =
+          EvictionAttributes.createLIFOEntryAttributes(100, EvictionAction.LOCAL_DESTROY);
+      dataRegionFactory.setEvictionAttributes(ea);
+      dataRegionFactory.setEnableAsyncConflation(true);
+
+      FixedPartitionAttributes fpa = FixedPartitionAttributes.createFixedPartition(PART2_NAME, 4);
+      PartitionAttributes pa = new PartitionAttributesFactory().setLocalMaxMemory(150)
+          .setRecoveryDelay(4).setTotalMaxMemory(200).setRedundantCopies(1)
+          .addFixedPartitionAttributes(fpa).create();
+      dataRegionFactory.setPartitionAttributes(pa);
+
+      dataRegionFactory.create(PR1);
+      createRegionsWithSubRegions();
+    });
+  }
+
+  @Test
+  public void describeRegionWithGatewayAndAsyncEventQueue() throws Exception {
```

Comment:
since we have only one tests here, let's put the system setup in this test. If any future tests needs to be added, it's up to them to put any common setup in a @Before tag. This would make it easy to spot what the test is doing.